### ISSUE TITLE
[Serious] State of active component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#584]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87615 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - States of User Interface Components (1.4.11.b) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/06/26 at 5:19 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

The state of an active component lacks 3 to 1 contrast ratio.
The selected tab has a light gray outline indicating the selection which would be especially difficult to see if you cannot see color (see grayscale screenshot).  See Settings User Experience video at 5:55.
To reproduce: select BagIt Profile from Settings, select New, accept the default APTrust, then select Create. On New profile based on APTrust (date) screen you will find About, Info, General, Manifests, Serialization, and Tag Files tabs.

## Recommendation

Rule : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

How To Fix: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

Reference: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast

Background: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

## Screenshots

![01_selected contrast](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87615/01_selected_contrast.png)
